test/framework/fileConversion.go

```diff
@@ -85,7 +130,7 @@ func qcow2Cmd(srcfile, tgtDir string) (string, error) {
 	return tgt, nil
 }
 
-func noopCmd(src, tgtDir string) (string, error) {
+func toNoop(src, tgtDir string) (string, error) {
```

Comment:
what do you think about this entire func begin:
```
func toNoop(src, tgtDir string) (string, error) {
     return copyIfNotPresent(src, tgtDir)
}
```

Reply:
Yup! Thanks!